### ISSUE TITLE
about softplus

### ISSUE THREAD

**Issue by ShellingFord221** [kumar-shridhar/PyTorch-BayesianCNN#17]
Hi, how do you implement softplus in your code? 
Thanks!

**Comment by ShellingFord221** [comment]
In /Image Recognition/main_Bayes.py, line 222:
        preds = F.softmax(outputs, dim=1)
I think you still use the results after softmax to get probabilities and calculate uncertainties.

**Comment by kumar-shridhar** [comment]
Softplus is used instead of ReLU as an activation function. 

For probabilities check the other repo: [link](https://github.com/kumar-shridhar/PyTorch-Softplus-Normalization-Uncertainty-Estimation-Bayesian-CNN)

**Comment by ShellingFord221** [comment]
Sorry, I didn't find any code in the repo above ...
